sensors/inject/imu.go

```diff
@@ -0,0 +1,39 @@
+package inject
```

Comment:
Refactoring: Split contents of `sensor_mock.go` into `inject/imu.go` and `inject/lidar.go` and changed the mock sensors into inject sensors (which they pretty much were already).

Reply:
Should this be inject/movement_sensor.go

Reply:
Discussed in person: The odometer and imu will have separate implementations. This also applies to the injected sensors.